Formula/jabba.rb

```diff
@@ -0,0 +1,26 @@
+class Jabba < Formula
+  desc "(cross-platform) Java Version Manager"
+  homepage "https://github.com/shyiko/jabba"
+  url "https://github.com/shyiko/jabba/archive/0.9.1.tar.gz"
+  sha256 "5e00ceec91afdea5ef2f2b5ffc764e552bb9837c1d69d7e1375136da46b15abd"
+  head "https://github.com/shyiko/jabba.git"
+
+  depends_on "go" => :build
+  depends_on "glide" => :build
+
+  def install
+    ENV["GOPATH"] = buildpath
+    dir = buildpath/"src/github.com/shyiko/jabba"
+    dir.install buildpath.children
+    cd dir do
+      ldflags = "-X main.version=#{version}"
+      system "glide", "install"
+      system "go", "build", "-ldflags", ldflags, "-o", "jabba"
+      bin.install "jabba"
+    end
+  end
+
+  test do
+    assert_equal version, `jabba --version`
```

Comment:
Any chance of a more detailed test here? We prefer to avoid testing help or version output if something slightly more complex is available.

Reply:
I understand, I tried to think of something, but couldn't. `jabba` doesn't do anything useful unless you install a Java version, which involves downloading remote files.

I tried to compare the test in a similar tool, but it appears equally trivial: https://github.com/Homebrew/homebrew-core/blob/master/Formula/nvm.rb#L38

Reply:
Not a problem; thanks for checking!

Reply:
Should be `shell_output("")` rather than backticks.

Reply:
We'll need a test that exercises and verifies the functionality of the software since this a new formula.

Reply:
> Should be shell_output("") rather than backticks.

Done.